commit 9909ee44c8c0815fbfed44fe8001c15ecc97e6f9
Author: Andreas Grueninger <andreas.grueninger@nomail.com>
Date:   Sat Oct 22 17:28:57 2016 +0200

    Fix: No special handling of gcc flags for Solaris
    
    In Solaris/Illumos sh is not bash. CONFIG_SHELL must be set with: export CONFIG_SHELL=/usr/gnu/bin/sh.

diff --git a/configure.ac b/configure.ac
index 01853b3..dc65eaa 100644
--- a/configure.ac.~1~	2017-07-06 03:04:17.000000000 +0000
+++ b/configure.ac	2017-09-17 08:52:08.441163906 +0000
@@ -1806,6 +1806,7 @@
         
 	# We had to eliminate -Wnested-externs because of libtool changes
         EXTRA_FLAGS="-fgnu89-inline
+		-fstack-protector-all				
 		-Wall
 		-Waggregate-return
 		-Wbad-function-cast
